app/src/main/res/drawable/continent_asia.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:viewportWidth="950.9475"
+    android:viewportHeight="787.0276"
+    android:width="950.94751dp"
+    android:height="787.02759dp">

```

Comment:
These should be `64dp` if I remember correctly. Bonus points for using `64px` in inkscape already and centering the drawing in it.